docs/user_guide/data_scientist_guide/federated_xgboost/secure_xgboost_user_guide.rst

```diff
@@ -4,69 +4,91 @@ NVFlare XGBoost User Guide
 
-1. Row split without encryption
-2. Column split without encryption
-3. Row split with HE (Requires at least 3 clients. With 2 clients, the other client's histogram can be deduced.)
-4. Column split with HE
+- **High performance** on structured data
+- **Explainability** of predictions
+- **Computational efficiency**
 
-When running with NVFlare, all communications in XGBoost are local and the messages are forwarded through NVFlare's communication.
+These examples use `DMLC XGBoost <https://github.com/dmlc/xgboost>`_, which provides GPU acceleration capabilities, distributed and federated learning support, and optimized gradient boosting implementations.
+
+Supported Training Modes
+-------------------------
+NVFlare supports federated training in the following 4 modes:
+
+1. Row split (horizontal) without encryption
+2. Column split (vertical) without encryption
+3. Row split (horizontal) with HE (Requires at least 3 clients. With 2 clients, the other client's histogram can be deduced.)
+4. Column split (vertical) with HE
+
+When running with NVFlare, all XGBoost communications are local and messages are forwarded through NVFlare's communication infrastructure.
 
 The encryption is handled in XGBoost by encryption plugins, which are external components that can be installed at runtime.
 
 Collaboration Modes and Data Split
 ===================================
-Essentially there are two collaboration modes: horizontal and vertical:
+Essentially, there are two collaboration modes: horizontal and vertical:
 
-- In horizontal case, each participant has access to the same features (columns) of different data samples (rows).
-  In this case, everyone holds equal status as "label owner".
-- In vertical case, each client has access to different features (columns) of the same data samples (rows).
-  We assume that only one is the "label owner" (or we call it as the "active party"), all other clients are "passive parties".
+- In the horizontal case, each participant has access to the same features (columns) of different data samples (rows).
+  In this case, everyone holds equal status as a "label owner".
+- In the vertical case, each client has access to different features (columns) of the same data samples (rows).
+  We assume that only one client is the "label owner" (also called the "active party"), while all other clients are "passive parties".
 
 Security Measures
 =================
+Security risks exist based on prior research [e.g. `SecureBoost <https://arxiv.org/abs/1901.08755>`_, `TimberStrike <https://arxiv.org/abs/2506.07605>`_] that exploits several types of information: sample-wise gradients for label recovery, gradient histograms for distribution recovery, and final model statistics for model inversion. We have three basic security categories:
```

Comment:
Suggests starts with 

"There are several categories of risks 
    * risk 1
    * risk 2
    * risk 3
see refs: research [e.g. `SecureBoost <https://arxiv.org/abs/1901.08755>`_, `TimberStrike <https://arxiv.org/abs/2506.07605>`_]